h2o-r/h2o-DESCRIPTION.template

```diff
@@ -38,7 +38,7 @@ Description: R interface for 'H2O', the scalable open source machine learning
     Deep Neural Networks (Deep Learning), Stacked Ensembles, Naive Bayes,
     Generalized Additive Models (GAM), ANOVA GLM, Cox Proportional Hazards, K-Means, PCA, ModelSelection,
     Word2Vec, as well as a fully automatic machine learning algorithm (H2O AutoML).
-License: Apache License (== 2.0)
+License: GNU Affero General Public License (== 3.0)
```

Comment:
I think it should be: (https://cran.rstudio.com/doc/manuals/r-devel/R-exts.html#Licensing)
```suggestion
License: AGPL-3
```

Reply:
Good point, thank you!